index.js

```diff
@@ -109,6 +109,8 @@ export function execa(file, args, options) {
 	spawned.kill = spawnedKill.bind(null, spawned.kill.bind(spawned));
 	spawned.cancel = spawnedCancel.bind(null, spawned, context);
 
+	const isCanceled = () => context.isCanceled || (parsed.options.signal ? parsed.options.signal.aborted : false);
```

Comment:
```suggestion
	const isCanceled = () => context.isCanceled || (parsed.options.signal && parsed.options.signal.aborted);
```

Reply:
It seems that when `signal` option is not using and context.isCanceled is false, 
isCanceled is `undefined` (It should be false in this case) (`false || undefined === undefined`)
Would it be better to set `signal` option's default value to false to handle this?

Reply:
Good point! :+1: